modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -158,11 +167,12 @@ public void onWrite(Iterator<CommandClosure<WriteCommand>> iterator) {
      */
     private void handleUpdateCommand(UpdateCommand cmd, long commandIndex) {
         storage.runConsistently(() -> {
-            BinaryRow row = cmd.getRow();
-            RowId rowId = cmd.getRowId();
+            BinaryRow row = cmd.rowBuffer() != null ? new ByteBufferRow(cmd.rowBuffer().toByteArray()) : null;
+            UUID rowUuid = cmd.rowUuid();
+            RowId rowId = new RowId(partitionId, rowUuid.getMostSignificantBits(), rowUuid.getLeastSignificantBits());
             UUID txId = cmd.txId();
-            UUID commitTblId = cmd.getCommitReplicationGroupId().getTableId();
-            int commitPartId = cmd.getCommitReplicationGroupId().getPartId();
+            UUID commitTblId = cmd.tablePartitionId().tableId();
```

Comment:
Why did you decided to rename it? I would leave the original name (commitReplicationGroupId)

Reply:
The main reason was to unify name of the field: it was named replicationGroupId in FinishTxCommand and commitReplicationGroupId in Update and UpdateAll commands. So they weren't uniform.
And return type wasn't uniform too: TablePartitionId in Update/UpdateAll and ReplicationGroupId in FinishTxCommand.

I decided to unify these names and chose name ```tablePartitionId``` to better reflect type of actual object that is transferred by commands.